content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,269 @@
+Contact`, :guilabel:`Education`, :guilabel:`Citizenship`, :guilabel:`Dependant`, and :guilabel:`Work
+Permit` information is entered. Fields are entered either using a drop-down menu, clicking a check
+box, or typing in the information.
+
+- :guilabel:`Private Contact`: Enter the :guilabel:`Address` for the employee. The selection can be
+  made with the drop-down menu. If the information is not available, type in the name for the new
+  address. To edit the new address, click the :guilabel:`External Link` button to open the address
+  form. On the address form, enter the necessary details, then click :guilabel:`Save`. Some other
+  information in the :guilabel:`Private Contact` section may auto-populate, if the address is
+  already listed in the drop-down menu.
+- :guilabel:`Marital Status`: Select either :guilabel:`Single`, :guilabel:`Married`,
+  :guilabel:`Legal Cohabitant`, :guilabel:`Widower`, or :guilabel:`Divorced` from the drop-down
+  menu.
+- :guilabel:`Emergency Contact`: Type in the name and phone number of the employee's emergency
+  contact.
+- :guilabel:`Education`: Select the highest level of education completed by the employee from the
+  :guilabel:`Certificate Level` drop-down menu. Options include :guilabel:`Graduate`,
+  :guilabel:`Bachelor`, :guilabel:`Master`, :guilabel:`Doctor`, or :guilabel:`Other`. Type in the
+  :guilabel:`Field of Study`, and the name of the :guilabel:`School` in the respective fields.
+- :guilabel:`Citizenship`: This section houses all the information relevant to the citizenship of
+  the employee. Some selections use a drop-down menu, as does the :guilabel:`Nationality (Country)`,
+  :guilabel:`Gender`, and :guilabel:`Country of Birth` sections. The :guilabel:`Date of Birth` uses
+  a calendar module to select the date. First, click on the name of the month, then the year, to
+  access the year ranges. Use the :guilabel:`< (left)` and :guilabel:`> (right)` arrow icons,
+  navigate to the correct year range, and click on the year. Next, click on the month. Last, click
+  on the day to select the date. Type in the information for the :guilabel:`Identification No`,
+  :guilabel:`Passport No`, and :guilabel:`Place of Birth` fields. If the employee is
+  :guilabel:`Disabled` or a :guilabel:`Nonresident`, click the check box next to the respective
+  fields.
+- :guilabel:`Dependant`: If the employee has any dependants, that information is entered here. Type
+  in the number of children the employee has, and check the boxes next to :guilabel:`Disabled
+  Children` and/or :guilabel:`Other Dependent People` if applicable.
+- :guilabel:`Work Permit`: If the employee has a work permit, enter the information in this section.
+  Type in the :guilabel:`Visa No` and/or :guilabel:`Work Permit No` in the corresponding fields.
+  Using the calendar module, select the :guilabel:`Visa Expire Date` to enter the expiration date.
+
+.. image:: new_employee/private-info.png
```

Comment:
This screenshot, as well as the others featuring forms in the doc, would serve better if the form fields were *filled in*. They can already see blank forms in their own databases, however what's really valuable is seeing a final result.

As is, these are fine, however something to consider for edits later for 15 and 16.

Reply:
While I hate to *not* do something, but since in a coupe months 14 will be gone, I will instead just make the image changes for 15 an d16.